package.json

```diff
@@ -2,10 +2,10 @@
   "name": "solenoid",
   "private": true,
   "devDependencies": {
+    "@biomejs/biome": "^1.9.4",
     "@types/bun": "^1.2.3",
-    "vitest": "^3.0.6",
     "typescript": "^5.7.3",
-    "@biomejs/biome": "^1.9.4"
```

Comment:
btw, biome is prettier replacement and needs to be set up. But I'll bring it back on my end.

Reply:
I didn't delete it, I think some rebasing issue.

Reply:
Oh it's not deleted. `bun` just did a sort of keys. It's up top now.